typings/balena-sdk.d.ts

```diff
@@ -1254,6 +1278,24 @@ declare namespace BalenaSdk {
 					applicationArchitecture: string,
 				): boolean;
 			};
+
+			_os: {
+				getAllOsVersions(
+					deviceTypes: string[],
+					appType?: ApplicationType,
+				): Promise<DeviceTypeOsVersions>;
```

Comment:
Let's just remove the `appType` so this doesn't include any app concept.
```suggestion
				getAllOsVersions(
					deviceTypes: string[],
				): Promise<DeviceTypeOsVersions>;
```